### ISSUE TITLE
1.7.3.1 editor "Cut" is not working, the grid content is still here

### ISSUE THREAD

**Issue by qch1842** [robotframework/RIDE#1870]
I have upgraded to 1.7.3.1, but found the Cut (Ctrl-X) is not working in Editor, the cell content will b kept while content is copied

**Comment by HelioGuilherme66** [comment]
On Windows 10 the selection of text is not visible. But the clipboard actions still work. You probably did not select the text when trying Cut. On Windows 7, Linux and MacOS the selection is visible.

**Comment by qch1842** [comment]
I experienced the issue on Win7 and Win10 both, in 1.5.2.1, it works.
![cut-not-working-1 7 3 1](https://user-images.githubusercontent.com/23063736/55594783-ca266180-56f5-11e9-9c30-178c4d05dd34.png)

**Comment by Nyral** [comment]
Fix from #1739 should also fix this issue.

**Comment by HelioGuilherme66** [comment]
@qch1842 You can see that fixed in current development version:
```
pip install -U https://github.com/robotframework/RIDE/archive/master.zip
```
